packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -601,15 +601,20 @@ export default {
           throw new Error(`An error has occurred: ${response.status}`)
         }
 
-        const path = pathUtil.join(this.currentPath, fileName)
         let resource
+        let path = pathUtil.join(this.currentPath, fileName)
+
         if (this.isPersonalLocation) {
+          path = buildWebDavFilesPath(this.user.id, path)
+          resource = await this.$client.files.fileInfo(path, DavProperties.Default)
+        } else if (this.isSpacesProjectLocation) {
+          path = buildWebDavSpacesPath(this.$route.params.spaceId, path)
           resource = await this.$client.files.fileInfo(path, DavProperties.Default)
         } else {
           resource = await this.$client.publicFiles.getFileInfo(
             path,
             this.publicLinkPassword,
-            DavProperties.PublicLink
+            DavProperties.Default
```

Comment:
Why not the public link dav properties? Those are all `Default` properties and some additional ones. Are they not needed?

Reply:
It's what you use everywhere. And by looking at the properties it looks to me that is the correct one.

Reply:
Well, no, for public files we usually use `DavProperties.PublicLink`. Looks to me like we should keep it that way, could you revert the line?

Reply:
You are correct! I took the opportunity to change it in the other places where it was requesting the default properties as well.